### PR TITLE
Upgrade to React Native 0.59.1

### PR DESCRIPTION
**Pull request by reime005**
Upgrade to React Native 0.59.1

### CODE REVIEW

**Comment by sibelius** [comment]
Can you fix CI status?

File: examples/basic/android/gradle.properties [hunk 1]
```diff
@@ -18,3 +18,4 @@
 # org.gradle.parallel=true
 
 android.useDeprecatedNdk=true
+android.enableAapt2=false
```
Comment (friederbluemle) [inline examples/basic/android/gradle.properties]: Just checking: Is this needed here? The default template of RN 0.59.1 doesn't have it. (`useDeprecatedNdk` is _not_ present as well). Also don't remove the newline character from the last line.

Comment (friederbluemle) [reply examples/basic/android/gradle.properties]: Based on this warning in the build log there is definitely no need to add this line here:

```
WARNING: The following project options are deprecated and have been removed: 
android.enableAapt2
This property has no effect, AAPT2 is now always used.
```

Comment (reime005) [reply examples/basic/android/gradle.properties]: sure, was just a try

**Comment by sibelius** [comment]
any guess why this is failing?

**Comment by sibelius** [comment]
💯

**Pull request merged**